### PR TITLE
Allow passing data-params-* to amp-youtube

### PR DESCRIPTION
**Pull request by muxin**
implements #2420

### CODE REVIEW

File: examples/youtube.amp.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <amp-youtube
             data-videoid="mGENRKrdoGY"
             layout="responsive"
+            data-param-autoplay=1
```
Comment (muxin) [inline examples/youtube.amp.html]: Do we add the data-param-* example here or create another youtube example?

Comment (jridgewell) [inline examples/youtube.amp.html]: This is fine.

**Comment by jridgewell** [comment]
LGTM.

File: extensions/amp-youtube/amp-youtube.md [hunk 2]
```diff
@@ -54,6 +54,16 @@ The Youtube video id found in every Youtube video page URL
...
 
+**data-param-***
+
+All `data-param-*` attributes will be added as query parameter to the youtube iframe src. This may be used to pass custom values through to youtube plugins, such as autoplay.
+
+Keys and values will be URI encoded. Keys will be camel cased.
+
+- `data-param-autoplay=1` becomes `&autoplay=1`
+
+See [Youtube Embedded Player Parameters](https://developers.google.com/youtube/player_parameters) for more parameter options for youtube.
```
Comment (mkhatib) [inline extensions/amp-youtube/amp-youtube.md]: Thanks for linking to the docs! 👍

**Pull request merged**